### ISSUE TITLE
Allow a Potential-Flow-Only Solution in HydroDyn

### ISSUE THREAD

**Issue by jjonkman** [OpenFAST/openfast#79]
There is a small bug preventing HydroDyn from functioning properly when a model has no strip-theory nodes (NJoints = NMembers = 0).  This is discussed more in the following forum topic: https://wind.nrel.gov/forum/wind/viewtopic.php?f=3&t=1227&p=8160.

As a workaround until this bug is fixed, one can set up a HydroDyn module with two joints and one member, but with all member properties and coefficients set to zero.

**Comment by ebranlard** [comment]
Since the number of members `Morison%NMembers` is 0, `Morison_Init` is not called, and then `y%Morison%DistribMesh` and `y%Morison%LumpedMesh` are not initialized (nor committed).
 
The error message occurs in `Hydrodyn.f90` around line 1440:
```fortran
CALL MeshMapCreate( y%Mesh,                m%y_mapped, m%HD_MeshMap%HD_P_2_WRP_P);
CALL MeshMapCreate( y%Morison%LumpedMesh,  m%y_mapped, m%HD_MeshMap%M_P_2_WRP_P);
CALL MeshMapCreate( y%Morison%DistribMesh, m%y_mapped, m%HD_MeshMap%M_L_2_WRP_P);
```
These lines can be replaced with:
```fortran
CALL MeshMapCreate( y%Mesh,                m%y_mapped, m%HD_MeshMap%HD_P_2_WRP_P);
IF ( y%Morison%LumpedMesh%Committed ) THEN 
    CALL MeshMapCreate( y%Morison%LumpedMesh,  m%y_mapped, m%HD_MeshMap%M_P_2_WRP_P);
ENDIF
IF ( y%Morison%DistribMesh%Committed ) THEN 
    CALL MeshMapCreate( y%Morison%DistribMesh, m%y_mapped, m%HD_MeshMap%M_L_2_WRP_P);
ENDIF
```


These if statements are already present elsewhere in the HydroDyn code so there is no more update required for `HydroDyn`.  I compared results with this patch or  with the hack mentioned by Jason (one member with no hydro coeffs) and the results matched. 

These meshes are yet used in `Fast_Subs.f90` and `Fast_Solver.f90`. 

 - For `Fast_Subs.f90`
No need for special handling since it's calling `MeshWrVTK` and `MeshWrBin` which are safe on meshes that are not committed. 

 - For `Fast_Solver.f90` 
In most places, things are safe:  `NNodes` is 0 for an uncommited mesh, `MeshCopy` works fine for an uncommitted mesh . There are yet a couple of if statement to add I believe. I'll discuss that with Jason and suggest a pull request

**Comment by rafmudaf** [comment]
Fixed by #258
